src/functionalTest/java/uk/gov/hmcts/darts/authentication/AuthenticationFunctionalTest.java

```diff
@@ -34,7 +35,7 @@ void shouldReturnUnauthorizedWhenSecuredEndpointIsCalledWithoutAuth() {
             .then()
             .extract().response();
 
-        assertEquals(401, response.statusCode());
+        assertFalse(response.getHeader("Location").isEmpty());
```

Comment:
Double negative makes this line abit hard to read - is there a better way? Do we know what the location should be or does it change each time? 
If not, maybe `assertTrue(response.getHeader("Location").isNotEmpty());` might be clearer?

Reply:
As discussed changed to assertNotnull.